### ISSUE TITLE
Accessibility: Megamenu is invisible to screenreaders

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/cgov-digital-platform#2227]
On the megamenu ul, the aria-hidden attribute is set to true, meaning that it cannot be seen by screenreaders.  The hidden attribute should be toggled alongside the megamenu's visibility but is hardcoded as true.

This issue exists in production.

Toggle the aria-hidden attribute with megamenu visibility.

<img width="486" alt="Screen Shot 2019-07-18 at 5 02 26 PM" src="https://user-images.githubusercontent.com/45469809/61491903-3b74e200-a97e-11e9-9ea4-f0d91433119c.png">

**Comment by arcepaul** [comment]
Ok, this one turned out not to be as big as I thought.  Going through with the screenreader, all that was really missing was the length of the list in the nav.  Removed the aria-hidden attribute because in the nav element, a list is expected anyway.

**Comment by markecramer** [comment]
Looks like the hidden attribute is working. Tested with Voice Over and I can hear the items in the megamenu. Thanks.
